### ISSUE TITLE
Remove capitalize() calls to avoid issues with other languages

### ISSUE THREAD

**Issue by th3hamm0r** [wagtail/wagtail#9219]
### Is your proposal related to a problem?

There are (only) 4 calls of `capitalize()` in wagtail's codebase and 3 of them actually lead to a bad wording in german, for example.

**1) `get_content_type_label()`:**
https://github.com/wagtail/wagtail/blob/bf3a054c04eb66a923bac538caa64165fef79131/wagtail/coreutils.py#L153-L163
I think the second call (fallback if no model exists) can be kept, but the first one, which uses `verbose_name`, should be removed.

This is used in the audit logs for example:
![wagtail_capitalize_calls_german](https://user-images.githubusercontent.com/468164/190669596-05cabfe5-8fe3-4fb7-a394-cf8894ac7783.png)
"Live-Update" (stored in `verbose_name`) is transformed to "Live-update", which looks like a mistake in german.

**2) Site settings:**
https://github.com/wagtail/wagtail/blob/bf3a054c04eb66a923bac538caa64165fef79131/wagtail/contrib/settings/models.py#L146-L147
https://github.com/wagtail/wagtail/blob/bf3a054c04eb66a923bac538caa64165fef79131/wagtail/contrib/settings/models.py#L201-L202

This results in the following:
![wagtail_site_settings_capitalize](https://user-images.githubusercontent.com/468164/190670976-d2d46dc9-0411-4e45-b5b0-d737dd626dcf.png)
This example also shows some other translation issues, which I've fixed in #9217 (and in transifex).

### Describe the solution you'd like

Can we remove the `capitalize()` calls where `verbose_name` is used, so it it can be properly translated?

**Comment by th3hamm0r** [comment]
Ok, "other languages", there aren't many, but for german it's an issue 😅
But even in english, e.g. for models that contain acronyms like "DNS" in their name, lowercasing everything and only uppercasing the first letter is not ideal I think.

Can we use something else than `capitalize()`? Only uppercasing the first character and leaving the rest untouched could work quite well I think.

**Comment by lb-** [comment]
Hey. I 100% agree that we should avoid using capitalise type functions in code. 

If you get a moment could you please add a note about this to the discussion about better supporting other languages. 

https://github.com/wagtail/wagtail/discussions/8017

I'll probably flag this as a bug as we already adopted similar changes in previous releases. See https://github.com/wagtail/wagtail/issues/7624

**Comment by gasman** [comment]
I never realised that `capitalize` lower-cases capital letters (other than the initial one) that were capitalised in the input! That seems counter-productive in _any_ language.

Looks like Django's `django.utils.text.capfirst` does what we want: https://github.com/django/django/blob/9c42933711667034c9335f408e1a8a53031e9865/django/utils/text.py#L14-L21

**Comment by lb-** [comment]
Note. The usage in https://github.com/wagtail/wagtail/blob/bf3a054c04eb66a923bac538caa64165fef79131/wagtail/coreutils.py#L153-L163

Is probably a different scenario - we are trying to make a useful label out of a model name. 

That one could probably use the capfirst approach or something a bit more nuanced, only the logic branch where we are processing the ContentType. 

The contrib settings and the logic in coreutils that reads the verbose name should not use any capitalisation - as it is provided by the code and hopefully the correct setting of verbose name.

**Comment by gasman** [comment]
> The contrib settings and the logic in coreutils that reads the verbose name should not use any capitalisation - as it is provided by the code and hopefully the correct setting of verbose name.

No, I don't think that's correct - in English, the `verbose_name` setting on a model should be uncapitalised so that it can be used in contexts like "Add a %s". The decision to capitalise it or not for different situations is made on a case-by-case basis, so any code that's taking the verbose_name with the intention of using it for a label or heading _should_ capitalise it at that point. The problem is that Python's `.capitalize()` method is unfit for that purpose.

(Presumably in German, where the model name _only_ ever appears in capitalised form, it's standard to write the `verbose_name` capitalised so that it appears that way even when English language rules wouldn't capitalise it - and a well-behaved capitalisation function would then leave it alone.)

**Comment by th3hamm0r** [comment]
Thank you both, `capfirst` is perfect I think.

I could change all 4 uses of `capitalize` to Django's `capfirst` on top of #9217, just to avoid a git conflict with the changed site settings code there. Ok?

**Comment by lb-** [comment]
Thank @gasman makes sense, there are probably cases where we do want to present this as a sentence with the model name somewhere in it (start or end) or stand-alone where sentence case also makes sense.

My only additional thought would be to move the [`capfirst` to the Django template](https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#capfirst) usage, unless that becomes too verbose, so it is closer to the context of being rendered.

On investigation, that seems to be the way we solve this kind of problem for many other similar situations like buttons or columns with data in them.

Examples
* wagtail/snippets/templates/wagtailsnippets/snippets/_header_with_history.html & pretty much all other snippets code where we refer to the model in a button/label/message/list.
* wagtail/users/templates/wagtailusers/groups/includes/formatted_permissions.html - formatting a permissions object (not a model but similar goal here)
* wagtail/contrib/forms/templates/wagtailforms/list_forms.html - forms listing where we output the form page's model name in the list. (although, this probably needs a more nuanced approach using the `get_content_type_label` util

**Comment by lb-** [comment]
@th3hamm0r  - as per your question - let's go with the adoption of the `capfirst` in your PR - makes sense, if in the template - great, if not that is fine also.

After more digging, I can see that we use `capfirst` in the Python code a lot also, even throughout the settings app, for essentially the same purpose.

**Comment by th3hamm0r** [comment]
I've just pushed my changes to #9217.
As noted in the commit, I was able to move the `capfirst` call to the template for the settings-part easily, since the only usages of those `__str__()` calls I've found were those for the `titletag` in `edit.html`.
I've not moved the calls from `get_content_type_label()` to keep the change simple for now, as it probably would have needed more changes to its usages, e.g. the following had to be wrapped again to apply `capfirst`, I think:
https://github.com/wagtail/wagtail/blob/bf3a054c04eb66a923bac538caa64165fef79131/wagtail/admin/views/reports/aging_pages.py#L56-L59)
